metatron/build.gradle

```diff
@@ -4,8 +4,11 @@ version '1.0-SPIKE'
 dependencies {
     compile configurations.dropwizard,
```

Comment:
```suggestion
    implementation configurations.dropwizard,
```

Reply:
There's a newer way of doing this that aligns with a recent PR that landed

Reply:
Can we leave this/these until the next iteration?  Chris will have to approve it a third time and Karol will have a fit about the extra commits that made it into the PR if I use the GitHub Commit suggestion button.

Reply:
👍 okay